cmd/performance-profile-creator/cmd/root.go

```diff
@@ -83,25 +94,25 @@ func Execute() {
 }
 
 type profileCreatorArgs struct {
-	powerConsumptionMode string
-	mustGatherDirPath    string
-	profileName          string
-	reservedCPUCount     int
-	splitCPUsAcrossNUMA  bool
-	disableHT            bool
-	rtKernel             bool
-	userLevelNetworking  bool
-	mcpName              string
+	powerConsumptionMode        string
+	mustGatherDirPath           string
+	profileName                 string
+	reservedCPUCount            int
+	splitReservedCPUsAcrossNUMA bool
+	disableHT                   bool
+	rtKernel                    bool
+	userLevelNetworking         bool
+	mcpName                     string
 }
 
 func init() {
 	args := &profileCreatorArgs{}
 	log.SetOutput(os.Stderr)
 	rootCmd.PersistentFlags().IntVarP(&args.reservedCPUCount, "reserved-cpu-count", "R", 0, "Number of reserved CPUs (required)")
 	rootCmd.MarkPersistentFlagRequired("reserved-cpu-count")
+	rootCmd.PersistentFlags().BoolVarP(&args.splitReservedCPUsAcrossNUMA, "split-reserved-cpus-across-numa", "S", true, "Split the Reserved CPUs across NUMA nodes")
```

Comment:
is this a synonim for packed/spread reserved CPU distribution?

Reply:
Yes that is correct! Do we want to call the argument `spread-reserved-cpus-across-numa` instead?